### PR TITLE
Rr/banner

### PR DESCRIPTION
**Pull request by RRicardotj**

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ html, body, #root {
   height: 100%;
   display: flex;
   flex-direction: column;
-  width: 100%;
+  width: 1440px;
```
Comment (abelosorio) [inline src/index.scss:10]: Fix width? Hmm why?

Comment (RRicardotj) [reply src/index.scss:10]: it's the style guide stablished figma

Comment (RRicardotj) [reply src/index.scss:10]: image

Comment (abelosorio) [reply src/index.scss:10]: what would happen if the screen is smaller than that?

Comment (RRicardotj) [reply src/index.scss:10]: Some components overflow

Comment (RRicardotj) [reply src/index.scss:10]: image

Comment (RRicardotj) [reply src/index.scss:10]: image

Comment (RRicardotj) [reply src/index.scss:10]: That because in the components styleguide on figma, like GroupCards and Banner have a fixeds width

Comment (abelosorio) [reply src/index.scss:10]: Hmm.. Ok, I think we can tackle this when we focus on mobile.

**Pull request merged**